Containerfile

```diff
@@ -1,3 +1,11 @@
+FROM registry.access.redhat.com/ubi9/toolbox@sha256:ac5b709f8674ee02db04cb4fc970edff1798ae974f9c0f523bd25b1b7c6bc862 AS builder
+
+WORKDIR /tmp
+
+RUN curl -sL https://github.com/jqlang/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/jq && chmod 755 /tmp/jq && \
+    curl -sL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz -o /tmp/openshift-client-linux.tar.gz && \
```

Comment:
These commands won't work with hermetic builds. I think you should remove that from the pipelines. :(